alpaca_data_cleaned.json

```diff
@@ -72782,7 +72782,7 @@
     {
         "instruction": "Generate alternative names for a new business.",
         "input": "",
-        "output": "Radiant Networking, Brilliant Bridge Solutions, Marvelous Markets, Brilliant Brands International, Limitless Logic, Illuminated Innovations, Splendid Streams, Inc."
+        "output": "- Radiant Networking\n- Brilliant Bridge Solutions\n- Marvelous Markets\n- Brilliant Brands International\n- Limitless Logic\n- Illuminated Innovations\n- Splendid Streams\n- Inc"
```

Comment:
Remove last `\n` to be  
`"output": "- Radiant Networking\n- Brilliant Bridge Solutions\n- Marvelous Markets\n- Brilliant Brands International\n- Limitless Logic\n- Illuminated Innovations\n- Splendid Streams Inc"`